styles/faas.css

```diff
@@ -1,388 +1,328 @@
   min-width: initial;
 }
 
-.faas-form-79.faas.column2 .faas-form label,
-.faas-form-63.faas.column2 .faas-form label,
-.faas-form-21.faas.column2 .faas-form label {
-  line-height: 20px;
+.resource-form .faas.column2.next .faas-form label {
+  width: fit-content;
+  padding-right: 20px;
+  margin-bottom: 8px;
+  display: flex;
+  align-items: center;
 }
 
-.faas-form-79.faas.column2 .faas-form input,
-.faas-form-79.faas.column2 .faas-form select,
-.faas-form-79.faas.column2 .faas-form textarea,
-.faas-form-63.faas.column2 .faas-form input,
-.faas-form-63.faas.column2 .faas-form select,
-.faas-form-63.faas.column2 .faas-form textarea,
-.faas-form-21.faas.column2 .faas-form input,
-.faas-form-21.faas.column2 .faas-form select,
-.faas-form-21.faas.column2 .faas-form textarea {
+.resource-form .faas .faas-form .row input,
+.resource-form .faas .faas-form textarea,
+.resource-form .faas .faas-form select {
   font-family: var(--body-font-family);
   font-size: var(--type-body-s-size);
+  padding: 6px 16px;
+  height: 40px;
   line-height: 20px;
 }
 
-.faas.faas-form-115 input:not([type=radio]):not([type=checkbox]):not([type=submit]) {
-  border-radius: 5px;
-  font-size: 16px;
-  line-height: 28px;
+.resource-form .faas.column1 .faas-form .row input {
   padding: 6px 16px;
 }
 
-.faas.next .faas-form .row.submit.next input {
-  color: #FFF;
-  background: var(--color-accent);
-  border: 2px solid var(--color-accent);
-  padding: 7px 18px 8px;
-  line-height: 20px;
-  border-radius: 20px;
-  font-size: 17px;
-  min-height: 21px;
-  margin: 0;
-  grid-column: 2 / 3;
-  width: 32px;
+.resource-form .faas.column2 .faas-form .row input {
+  padding: 9px 16px;
 }
 
-.faas-form-79.faas.column2 .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]),
-.faas-form-79.faas.column2 .faas-form select,
-.faas-form-79.faas.column2 .faas-form textarea,
-.faas-form-63.faas.column2 .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]),
-.faas-form-63.faas.column2 .faas-form select,
-.faas-form-63.faas.column2 .faas-form textarea,
-.faas-form-21.faas.column2 .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]),
-.faas-form-21.faas.column2 .faas-form select,
-.faas-form-21.faas.column2 .faas-form textarea {
+.resource-form .faas .faas-form select,
+.resource-form .faas .faas-form textarea,
+.resource-form .faas .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]) {
   border: 1px solid #6E6E6E;
   border-radius: 4px;
   background-color: #fff;
   box-shadow: none;
   padding: 9px 16px;
 }
 
-.faas-form-79.faas.column2 .faas-form select,
-.faas-form-63.faas.column2 .faas-form select,
-.faas-form-21.faas.column2 .faas-form select {
+.resource-form .faas.column1 .faas-form select,
+.resource-form .faas.column1 .faas-form textarea {
+  padding: 6px 16px;
+}
+
+.resource-form .faas.column2 .faas-form select,
+.resource-form .faas.column2 .faas-form textarea {
+  padding: 9px 16px;
+}
```

Comment:
I believe I've seen these paddings before, lines 107 and 111 respectively. Can they be consolidated?

Reply:
If I increase the specificity, but then it cascades down to other rules..
~~~css
.resource-form .faas.column1 .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]),
.resource-form .faas.column1 .faas-form select,
.resource-form .faas.column1 .faas-form textarea {
  padding: 6px 16px;
}

.resource-form .faas.column2 .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]),
.resource-form .faas.column2 .faas-form select,
.resource-form .faas.column2 .faas-form textarea {
  padding: 9px 16px;
}
~~~

Reply:
Why do you need to increase the specificity in order to consolidate the lines?

Reply:
Because without increase specificity the linter complains about the order.

For example:
![image](https://github.com/adobecom/bacom/assets/681036/285ea776-fe6c-48ee-b116-7ab1e7195f78)

~~~
Expected selector ".resource-form .faas.column2 .faas-form .row input" to come before selector ".resource-form .faas .faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit])" (no-descending-specificity)Stylelint[no-descending-specificity](https://stylelint.io/user-guide/rules/list/no-descending-specificity)
~~~
~~~
Expected selector ".resource-form .faas .faas-form select" to come before selector ".resource-form .faas.column1 .faas-form select" (no-descending-specificity)Stylelint[no-descending-specificity](https://stylelint.io/user-guide/rules/list/no-descending-specificity)
~~~

Reply:
Ok, then leaving them like they were seems best